### PR TITLE
Capacitor support

### PR DESCRIPTION
**Pull request by yong-asial**

### CODE REVIEW

File: src/monaca.js [hunk 1]
```diff
@@ -1192,7 +1196,7 @@
               if (!platformContent.is_versionname_valid) {
                 return 'Version name is invalid.';
               }
-              if (buildType === 'release' && (!checkForMinimumRequirements && !platformContent.has_keysetting || checkForMinimumRequirements && !platformContent.has_keystore)) {
+              if (buildType === 'release' && (!checkForMinimumRequirements && !platformContent.has_keysetting || !checkForMinimumRequirements && !platformContent.has_keystore)) {
```
Comment (as-hirose) [inline src/monaca.js:1165]: Is this change intentional?

**Review by as-hirose — COMMENTED** [review]
@yong-asial I have added a question.

File: src/monaca.js (hunk 1 above)
Comment (yong-asial) [reply src/monaca.js:1165]: yes, it was a bug.

Comment (as-hirose) [reply src/monaca.js:1165]: Got it.

**Review by as-hirose — APPROVED** [review]
